connect-and-run.md

```diff
@@ -0,0 +1,213 @@
+| **SAS Path** | Path to SAS Executable on the server | Must be a fully qualified path on the SSH server to a SAS executable |
+
+### Add New SAS 9.4 Remote Profile
+Open the command palette (`F1`, or `Ctrl+Shift+P` on Windows or Linux, or `Shift+CMD+P` on OSX). After executing the `SAS.addProfile` command, select the SAS 9.4 (remote) connection type and complete the prompts (using values from the preceeding table) to create a new profile.
+
+
+## Delete Connection Profile
+
+After executing the `SAS.deleteProfile` command:
+
+1. Select profile to delete from the list of profiles
+2. A notification message will pop up on successful deletion
+
+## Switch Current Connection Profile
+
+After executing the `SAS.switchProfile` command:
+
+1. If no profiles can be found, the extension will ask to [create a new profile](#add-new-sas-profile)
+2. Select profile to set active from the list of profiles
+3. The StatusBar Item will update to display the name of the selected profile
+
+## Update Connection Profile
+
+Update profile gives the ability to modify existing profiles, including updating from password to token flow and vice versa.
+
+After executing the `SAS.updateProfile` command:
+
+1. Select profile to update from the list of profiles
+2. Complete the prompts to update profile
+
+To update the name of a profile, please delete and recreate it.
+
+#### Notes:
+
+- There is a potential issue with switching between multiple profiles on Windows. For more information, see [#215](https://github.com/sassoftware/vscode-sas-extension/issues/215).
+
+
+# Running SAS Code
+
+After configuring the SAS extension for your SAS environment, run your SAS program and view the log and results. The connection set up and process is different for SAS Viya and SAS 9. Each is explained in detail below.
+
+## SAS Viya 
+To run a SAS program connection to a SAS Viya instance:
+
+1. Click the running man icon in the upper right corner of your SAS program window.
+2. For a secure connection to SAS Viya we use an authorization code for authentication. Complete these steps to connect.
+
+   2.1. VS Code may prompt you that the extension wants to sign in. Click 'Allow'.
+
+   2.2. VS Code may prompt you about opening an external web site. Click 'Open'.
+
+   2.3. This will take you to a SAS Logon prompt. Log in with your SAS credentials.
+
+   2.4. SAS returns an authorization code. Copy this code.
+
+   2.5. Paste the authorization code in VS Code where indicated at the top of the screen.
+
+
+**Note:** Your sign in status will persist in VS Code. You can view it and sign out from VS Code's `Accounts` menu.
+
+3. VS Code connects to SAS and runs the code.
+4. The results are displayed in the application.
+5. The SAS output log and error information are displayed in the applicaiton.
+
+<img src="doc/images/runCode2.png"/>
+
+**Note:** Your sign in status will persist in VS Code. You can view it and sign out from VS Code's `Accounts` menu.
```

Comment:
duplicated with line 185 above

Reply:
Removing the line 185 version of the text.